python/cugraph/dask/traversal/sssp.py

```diff
@@ -0,0 +1,126 @@
+# Copyright (c) 2019-2020, NVIDIA CORPORATION.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+
+from dask.distributed import wait, default_client
+from cugraph.dask.common.input_utils import get_distributed_data
+from cugraph.structure.shuffle import shuffle
+from cugraph.dask.traversal import mg_sssp_wrapper as mg_sssp
+import cugraph.comms.comms as Comms
+import cudf
+import dask_cudf
+
+
+def call_sssp(sID,
+              data,
+              num_verts,
+              num_edges,
+              partition_row_size,
+              partition_col_size,
+              vertex_partition_offsets,
+              start):
+    wid = Comms.get_worker_id(sID)
+    handle = Comms.get_handle(sID)
+    return mg_sssp.mg_sssp(data[0],
+                           num_verts,
+                           num_edges,
+                           partition_row_size,
+                           partition_col_size,
+                           vertex_partition_offsets,
+                           wid,
+                           handle,
+                           start)
+
+
+def sssp(graph,
+         source):
+
+    """
+    Find the distances and predecessors for a breadth first traversal of a
+    graph.
+    The input graph must contain edge list as  dask-cudf dataframe with
+    one partition per GPU.
+
+    Parameters
+    ----------
+    graph : cugraph.DiGraph
+        cuGraph graph descriptor, should contain the connectivity information
+        as dask cudf edge list dataframe(edge weights are not used for this
+        algorithm). Undirected Graph not currently supported.
```

Comment:
"Undirected Graph not currently supported."
Could we add that undirected graphs should be passed as Digraph with edges in both directions (apply to other MG APIs as well).